requirements.win64.conda.yml

```
@@ -31,7 +31,7 @@ dependencies:
   - pytorch>=1.9.0
   - pyyaml>=5.1
   - requests
-  - scikit-image==0.18.3
+  - scikit-image==0.19.1
```

```suggestion
  - scikit-image>=0.19.1
```

We need to test with `>=19.1`